### ISSUE TITLE
Feature Request - warn when an invalid openApiOperationId is specified

### ISSUE THREAD

**Issue by jpjpjp** [apideck-libraries/portman#632]
I recently discovered that some of the tests I thought I had written weren't being created.  After some investigation, I realized that the OperationId I specified didn't match what was in my spec.

In my case I had a variation test that looked something like this:
```yaml
version: 1.0
tests:
  integrationTests:
    - name: /categories tests
       operations:
         - openApiOperationId: updateCategoryById
            openApiResponse: '200'
            variations:
```
Since my spec file didn't have an updateCategoryById none of the tests specified in this section got generated.

It would be nice if Portman would generate a warning, or even outright fail when it can't find a specified OpenAPIOperationId

Thanks for your consideration!

**Comment by thim81** [comment]
hi @jpjpjp 

Would this be something you had in mind for the warning?
<img width="968" alt="image" src="https://github.com/user-attachments/assets/6abc04bb-33a7-4b4f-a08e-4871875317f9">

**Comment by jpjpjp** [comment]
Yes.  This would be really helpful to catch mistakes!

**Comment by thim81** [comment]
hi @jpjpjp 

We just released version 1.30.5 of Portman, which includes the warning when a target does not exist in the OpenAPI spec.
Let us know if you have improvements or ideas.

Closing this feature request as completed.
